build/scripts/build.ps1

```diff
@@ -496,7 +496,7 @@ function Test-XUnit() {
 # Deploy our core VSIX libraries to Visual Studio via the Roslyn VSIX tool.  This is an alternative to 
 # deploying at build time.
 function Deploy-VsixViaTool() { 
-    $vsixDir = Get-PackageDir "roslyntools.microsoft.vsixexpinstaller"
+    $vsixDir = Get-PackageDir "RoslynTools.Microsoft.VSIXExpInstaller"
```

Comment:
Why the casing change here? The lower case form is more apporpriate as it's how NuGet lays it out on disk.

Reply:
Get-VersionCore is now case sensitive against the package name as it exists in NuGet.

Reply:
If you want the names to match what appears on disk, we need to modify `Get-PackageDir` to correct the case in the invocation of `Join-Path`. Let me know if you want me to make this change.

Reply:
Probaly should just change the implementation of Get-PackageDir to lower the case it's provided. That's the better fix and hides the implementation detail of NuGet.